### PR TITLE
Revert "Fix/asan"

### PR DESCRIPTION
**Pull request by timosachsenberg**
Reverts OpenMS/OpenMS#5947
Sorry have to revert. Still fails in mac with segfault.

### CODE REVIEW

File: src/openms/source/ANALYSIS/SVM/SVMWrapper.cpp [hunk 1]
```diff
@@ -581,16 +581,15 @@ namespace OpenMS
     vector<Size> indices;
     vector<Size>::iterator indices_iterator;
 
-    // Note: delete the structures, but keep the nodes
-    for (Size k = 0; k < problems.size(); k++)
+    for (Size i = 0; i < problems.size(); ++i)
     {
-      LibSVMEncoder::destroyProblem(problems[k], false);
```
Comment (hroest) [inline src/openms/source/ANALYSIS/SVM/SVMWrapper.cpp:587]: only revert this?

File: src/openms/source/ANALYSIS/SVM/SVMWrapper.cpp [hunk 2]
```diff
@@ -1089,7 +1084,6 @@ namespace OpenMS
           delete training_data_ul[k]; // delete individual objects
         }
         delete[] training_data_ul; // delete array of pointers
-        SVMWrapper().createRandomPartitions(problem_ul, 1, partitions_ul); // cleanup ...
```
Comment (hroest) [inline src/openms/source/ANALYSIS/SVM/SVMWrapper.cpp:1092]: and this?

**Comment by timosachsenberg** [comment]
sure make sense to keep the others. can you open a new PR with just those changes or via direct edit in the browser then we can close this one

**Comment by hroest** [comment]
I reverted the edits in SVMWrapper: bc87d11b6e248427ed6706e31b6965e30

**Comment by timosachsenberg** [comment]
strange... https://cdash.openms.de/viewTest.php?onlyfailed&buildid=21603
should already contain your rever commit.
maybe the SVMWrapper_test.cpp itself?

**Pull request closed without merging**